packages/compass-connection-import-export/src/components/export-modal.tsx

```diff
@@ -13,10 +13,8 @@ import { SelectTable } from './select-table';
 import type { ImportExportResult } from '../hooks/common';
 import { useOpenModalThroughIpc } from '../hooks/common';
 import { useExportConnections } from '../hooks/use-export';
-import type {
-  ConnectionInfo,
-  ConnectionStorage,
-} from '@mongodb-js/connection-storage/renderer';
+import type { ConnectionInfo } from '@mongodb-js/connection-info';
```

Comment:
IIUC what this is, it doesn't make much sense in compass outside of connection-storage. Maybe we can re-export the type so that we don't need to depend on two packages every time we need to use connection storage?